web/src/Routes.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react'
 import { Route, Switch, BrowserRouter as Router } from 'react-router-dom'
-import JobIndex from './screens/test'
-import RoleShow from './screens/roleShow'
+import JobIndex from './screens/JobIndex'
+import RoleShow from './screens/RoleShow'
 
 const Sample = () => (<h1>Sample Functional Component</h1>)
 
 export default () => (
   <Router>
     <Switch>
       <Route exact path="/" component={ JobIndex } />
-      <Route exact path="/front-end" component={ RoleShow } />
-      <Route exact path="/back-end" component={ Sample } />
-      <Route exact path="/qa" component={ Sample } />
+      <Route exact path="/jobs/:category" component={ RoleShow } />
```

Comment:
This is better, but "front-end", "back-end", and "qa" aren't mutually exclusive, and aren't the only options. Let's handle this with tags in the DB, and query strings.